Create Japanese Translation documents for Workshop

yoshioterada opened the pull request:
Create Japanese Translation documents for Workshop

yoshioterada committed:
Created Japanese Translation Documents for Workshop

Created Japanese Translation Documents for Workshop

yoshioterada committed:
Create Japanese Translation documents for Workshop

Create Japanese Translation documents for Workshop

yoshioterada committed:
Deleted wrong explanation

Deleted wrong explanation

yoshioterada committed:
Updated Markdown missing

Updated Markdown missing

yoshioterada committed:
Modified Markdown expression

Modified Markdown expression

yoshioterada committed:
Upload Image files

Upload Image files

yoshioterada committed:
Added AI Workshop

Added AI Workshop

yoshioterada committed:
Modified Link for AI

Modified Link for AI

dingmeng-xue commented:
@yoshioterada , thanks for contribution. Could you add somebody who can read Japanese for review? Then, we can merge it.

yoshioterada committed:
Create README.md

Create README.md

yoshioterada committed:
Update README.md

Modified from "Azure Active Directory" to "Microsoft Entra ID"

yoshioterada committed:
Update README.md

コマンドの実行順によるエラーが発生したため、順番を訂正

yoshioterada committed:
Update README.md

Log Analytics ワークスペースが作成されていなかったため

yoshioterada committed:
Update README.md

正しく動作するように修正

yoshioterada committed:
Update README.md

Keyvault が作成されていないのに Key Vault の操作があったため、
Keyvault の作成を追加

yoshioterada committed:
Update README.md

ルーティング設定のファイルの場所が異なるため、ディレクトリを移動。
Portal API のサンプル画面が表示されていなかったため、画面ダンプを追加

yoshioterada committed:
Update README.md

Java アプリのデプロイに失敗していたため、正しい引数に修正

yoshioterada committed:
Update README.md

説明を追記

yoshioterada committed:
Update README.md

PostgreSQL のパスワード設定などを追記

yoshioterada committed:
Update README.md

Update README.md

yoshioterada committed:
Update for intermediate

Update for intermediate

yoshioterada committed:
Update README.md

Modified Invalid URL for the image

yoshioterada committed:
Update README.md

Added OpenAI Environment Variables

yoshioterada committed:
Update README.md

Modified Environment Variable name

yoshioterada committed:
Update README.md

Deleted English explanations

yoshioterada committed:
Update README.md

Modified the Connection String for PostgreSQL from C#

yoshioterada committed:
Modified Connection Strings

Modified Connection Strings

yoshioterada committed:
Update README.md

Modified the style with more visible

yoshioterada committed:
Update README.md

Update README.md

yoshioterada committed:
Update README.md

Update README.md

yoshioterada committed:
Merge branch 'Azure-Samples:Azure' into japanese-translation

yoshioterada committed:
Update README.md

説明を修正

yoshioterada committed:
Update README.md

Log Analytics は ARM テンプレートで作成されるため、作成方法は削除

yoshioterada committed:
Modified LR

Modified Line separator

yoshioterada committed:
Modified to run the script before az login

Modified to run the script before az login

yoshioterada committed:
Deleted Invalid command

Deleted Invalid command

yoshioterada committed:
Modified Invalid explanation for change the KEY_VAULT settings

Modified Invalid explanation for change the KEY_VAULT settings

yoshioterada committed:
Added BP_JVM_VERSION arguments

Added BP_JVM_VERSION arguments

yoshioterada committed:
Deleted invalid character and english statement

Deleted invalid character and english statement

yoshioterada committed:
Modified operation for run the scripts

Modified operation for run the scripts

yoshioterada committed:
Deleted command which no need to execute

Deleted command which no need to execute

m-moris commented:
@yoshioterada 

Thanks for writing the Japanese documentation of the detailed procedure.
I attended this workshop and reviewed these documents.

suzukiyuhsuke commented:
@yoshioterada,
Sorry my late response, I'd think that chapter-14 has a incompleted section that the "Add a Instrumation Key to KeyValut" is missing detailed instructions.

yoshioterada commented:
> @yoshioterada, Sorry my late response, I'd think that chapter-14 has a incompleted section that the "Add a Instrumation Key to KeyValut" is missing detailed instructions.

Thank you so much for your feedback @suzukiyuhsuke san. I will check and modify it.

yoshioterada committed:
Added detail instructions

Added detail instructions.
This modification was coming from @suzukiyuhsuke san's feedback.

yoshioterada committed:
Added explanation

Added explanation

yoshioterada commented:
Dear @dingmeng-xue san

Yesterday, 
we had done the ASA-E workshop in Japan using this translated documents.
During the workshop, I got some feedbacks from the participants directlly.
And I immediately corrected any mistakes in wording or operational problems on the spot.

So now, all of the issue fixed and it is possible to merge it now.
So please merge this to the original.

dingmeng-xue committed:
Merge branch 'Azure' into japanese-translation

yoshioterada committed:
Merge branch 'Azure-Samples:Azure' into japanese-translation

yoshioterada committed:
modified the directory name from ja to ja-jp

From the request from Dingmeng-san, I modified the directory name from ja to ja-jp

yoshioterada committed:
I moved the ja-jp directory under the azure-spring-apps-enterprise directory

I moved the ja-jp directory under the azure-spring-apps-enterprise directory

dingmeng-xue committed:
update home page

dingmeng-xue commented on azure-spring-apps-enterprise/ja-jp/README.md:
```diff
@@ -0,0 +1,70 @@
+## Tanzuアプリケーションプラットフォーム(TAP)へのアプリのデプロイ
+
+Tanzuアプリケーションプラットフォーム(TAP) を使用すると、多言語アプリを CNCF に準拠する Kubernetes クラスター上に、簡単に構築して実行できます。
+[tanzu-application-platform/README-jp.md](./tanzu-application-platform/README-jp.md) の記載内容に従い、アプリケーションを TAP にデプロイします。
```
@yoshioterada , what's the correct link to this file? I cannot find it.

dingmeng-xue committed:
copy pictures

dingmeng-xue committed:
Update links

dingmeng-xue committed:
update links

dingmeng-xue committed:
update links

dingmeng-xue committed:
update links

dingmeng-xue committed:
update links

dingmeng-xue committed:
update links

dingmeng-xue committed:
remove pictures

dingmeng-xue committed:
update links

dingmeng-xue committed:
update links

yoshioterada committed:
Update README.md

Update Japanese sentence more natural by Japanese people.

yoshioterada committed:
Update README.md

I added line separator

yoshioterada committed:
Update README.md

I would like to add line separator

dingmeng-xue committed:
Merge branch 'Azure' into japanese-translation
